Makefile

```diff
@@ -46,18 +46,18 @@ endif
 #
 
 smoke-test:	test-clean-up	build_test
-	$(TEST_BIN) --spu ${DEFAULT_SPU} --produce-iteration ${DEFAULT_ITERATION} --local ${TEST_LOG} ${SKIP_CHECK} ${EXTRA_ARG}
+	$(TEST_BIN) smoke --spu ${DEFAULT_SPU} --produce-iteration ${DEFAULT_ITERATION} --cluster-type local ${TEST_LOG} ${SKIP_CHECK} ${EXTRA_ARG}
```

Comment:
Instead of `--local` it is `--cluster-type local`. And for k8 is is `--cluster-type k8`. I thought offering the options in this manner was clearer to use.

Reply:
it's more typing and we don't have other cluster type so it's not offer advantage.

Reply:
Changed back to `--local`